### PR TITLE
Bug/utf 8 encryption decryption issue

### PR DESCRIPTION
**Pull request by vijayshiyani**

### CODE REVIEW

File: test/encryption-decryption.test.ts [hunk 1]
```diff
@@ -32,13 +32,13 @@ describe('aes-256-gcm', () => {
     }
   });
 
-  Object.values(CipherStrategy).forEach((strategy) => {
-    Object.values(SerializationFormat).forEach((version) => {
+  Object.values([CipherStrategy.AES_GCM]).forEach((strategy) => {
+    Object.values([SerializationFormat.latest_version]).forEach((version) => {
```
Comment (zbarbuto) [inline test/encryption-decryption.test.ts]: I think revert the above two? This was just to reduce the number of tests, right?

Comment (vijayshiyani) [reply test/encryption-decryption.test.ts]: Yes, will fix it, soon. Thanks

**Comment by hongfeiyang** [comment]
can we please don't put my name here 😂, I will give you some other characters for testing, for example, below are translation for 'this is a test':
这是一个测试
이것은 테스트입니다
これはテストです
هذا اختبار
यह एक परीक्षण है
Это проверка
ഇതൊരു പരീക്ഷണമാണ്
ఇది ఒక పరీక్ష

**Comment by vijayshiyani** [comment]
> can we please don't put my name here 😂, I will give you some other characters for testing, for example, below are translation for 'this is a test':
> 这是一个测试
> 이것은 테스트입니다
> これはテストです
> هذا اختبار
> यह एक परीक्षण है
> Это проверка
> ഇതൊരു പരീക്ഷണമാണ്
> ఇది ఒక పరీక్ష

👍 fixed

**Review by brewdium — APPROVED** [review]

**Review by zbarbuto — APPROVED** [review]

**Pull request merged**